network/pcap.ksy

```diff
@@ -34,6 +34,8 @@ types:
         contents: [0xd4, 0xc3, 0xb2, 0xa1]
       - id: version_major
         type: u2
+        valid:
+          max: 2
```

Comment:
@armijnhemel Does the specification support major versions `0` and `1`?

Reply:
> I would expect very few version 0 and 1 files to exist. I wouldn't object to only making `2` valid and ignore `0` and `1`.

Yep, I'd go for that - we declare that we're only confident that the spec works with major version 2. If there still is some PCAP file version 0 or 1 (which I doubt, given that you write "_Version 2.4 (so major version 2) has been used since at least 1998._"), at least we'll be able to quickly identify it and adapt the spec accordingly, if it makes sense.

```suggestion
        valid:
          eq: 2
```